bot.js

```diff
@@ -0,0 +1,132 @@
+const mineflayer = require("mineflayer");
+const ipc = require("node-ipc");
+const v = require("vec3");
+
+const botId = process.argv[2];
+const server = process.argv[3];
+const port = process.argv[4];
+const username = process.argv[5] ? process.argv[5] : "bot_" + botId;
+const password = process.argv[6] ? process.argv[6] : null;
+
+ipc.config.id = "parent";
+ipc.config.appspace = "";
+ipc.config.silent = true;
+ipc.connectTo("parent", "/tmp/mineflayer.sock", function () {
+  parent = ipc.of.parent;
+  ipc.log("## connected to parent ##".rainbow, ipc.config.delay);
+
+  parent.on("data", function (data) {
+    switch (data.action) {
+      case "stop":
+        stop();
+        break;
+      case "status":
+        sendCoordinates();
+        sendStatus();
+        break;
+      case "getCoords":
+        sendCoordinates();
+        break;
+      case "say":
+        console.log("[INFO]: " + data.message);
+        bot.chat(data.message);
+        break;
+      case "getHealth":
+        sendHealth();
+        break;
+      case "getHunger":
+        sendHunger;
+        break;
+      case "place":
+        break;
+      case "attack":
+        break;
```

Comment:
Not used anywhere

Reply:
Isn't the health and hunger used?

Reply:
`getHealth` and `getHunger` are used, but the other three aren't